### PR TITLE
Add FormatHandle for debug object name

### PR DESCRIPTION
**Pull request by locke-lunarg**
Added FormatHandle to show handle's name & ID or only ID.
Test the function in core_validation, object_tracker, 
parameter_validation layer.

### CODE REVIEW

**Comment by mark-lunarg** [comment]
@locke-lunarg, Travis found some clang-formatting issues -- you can see what they are by looking at the details and then at the check_format console output.  Those fixes should get folded into that same commit -- after that, just force-push to the same branch and Github will rerun the CI checks automatically.

**Comment by locke-lunarg** [comment]
> @locke-lunarg, Travis found some clang-formatting issues -- you can see what they are by looking at the details and then at the check_format console output. Those fixes should get folded into that same commit -- after that, just force-push to the same branch and Github will rerun the CI checks automatically.

fix it.

**Comment by locke-lunarg** [comment]
> @locke-lunarg, Travis found some clang-formatting issues -- you can see what they are by looking at the details and then at the check_format console output. Those fixes should get folded into that same commit -- after that, just force-push to the same branch and Github will rerun the CI checks automatically.

Thank you. The format problem fixed, and I pushed new code.

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -1532,6 +1532,91 @@ TEST_F(VkLayerTest, DebugMarkerNameTest) {
...
+
+    VkCommandBufferBeginInfo cb_begin_Info = {};
+    cb_begin_Info.sType = VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO;
+    cb_begin_Info.flags = VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT;
+    vkBeginCommandBuffer(commandBuffer, &cb_begin_Info);
+
+    const VkRect2D scissor = {{-1, 0}, {16, 16}};
+    const VkRect2D scissors[] = {scissor, scissor};
+
+    // Test parameter_valiation layer
```
[the remark this answers is not in the document]
Comment (locke-lunarg) [reply tests/layer_validation_tests.cpp]: Thank you. Fix it.

File: layers/vk_layer_logging.h [hunk 2]
```diff
@@ -138,8 +138,9 @@ typedef struct _debug_report_data {
     }
 
     std::string FormatHandle(uint64_t h) const {
-        std::string ret = "0x";
-        ret.append(std::to_string(h));
+        char uInt64String[64];
```
Comment (mark-lunarg) [inline layers/vk_layer_logging.h]: Hey Locke, for the most part we try to follow the Google Style Guidelines, including variable and function naming.  So local variables should be lower_camel_case.

Comment (locke-lunarg) [reply layers/vk_layer_logging.h]: Thank you. Fixed it. I though "int" was the second word.

**Pull request merged**